src/pages/styledComponents.js

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/megwali/spacex/pullRequest?prid=7693722)
```suggestion
import styled from "styled-components";
```